### PR TITLE
user layout and admin layout

### PR DESCRIPTION
**Pull request by vutuankiet4599**
## Related Tickets:

-   [Task #67868](https://edu-redmine.sun-asterisk.vn/issues/67868)
-   [Task #67869](https://edu-redmine.sun-asterisk.vn/issues/67869)

## WHAT this PR does?

- Tạo layout view cho 2 role là user và admin
- Sửa lại test của breeze sau khi rebase master
## Notes Impacted Areas:

-   N/A.

### Test

image

### Layout view admin

image

### Layout view user

image

### CODE REVIEW

**Comment by tuanngoc1601** [comment]
ok

**Comment by vutuankiet4599** [comment]
ready

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@ yarn-error.log
 /.vscode
 /public/css
 /public/js
+/public/mix-manifest.json
+/public/fonts
```
Comment (ghost) [inline .gitignore]: eof

File: resources/views/layouts/admin.blade.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                </main>
+            </div>
+            
+        </div>
+
+        <script src="{{ asset('js/adminLayout.js') }}">
+            
+        </script>
+    </body>
+</html>
```
Comment (ghost) [inline resources/views/layouts/admin.blade.php]: eof

File: resources/views/test.blade.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="py-12">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+            <div class="bg-white overflow-hidden shadow-sm sm:rounded-lg">
+                <div class="p-6 bg-white border-b border-gray-200">
+                    You're logged in!
+                </div>
+            </div>
+        </div>
+    </div>
+</x-admin-layout>
```
Comment (ghost) [inline resources/views/test.blade.php]: eof

**Comment by vutuankiet4599** [comment]
ready

**Pull request merged**